docs/data_driven_testing.adoc

```diff
@@ -57,6 +57,28 @@ Data tables must have at least two columns. A single-column table can be written
 include::{sourcedir}/datadriven/DataSpec.groovy[tag=single-column]
 ----
 
+A sequence of two or more underscores can be used to separate multiple data tables.
```

Comment:
Are they multiple data tables, or is it just one divided into multiple sections?

Reply:
I'm not entirely sure what you mean be "one divided into multiple sections".
```groovy
a | _
1 | _
7 | _
0 | _
__

b | c
1 | 2
3 | 4
5 | 6
```
is effectively the same as
```groovy
a | b | c
1 | 1 | 2
7 | 3 | 4
0 | 5 | 6
```
or
```groovy
a << [1, 7, 0]
b << [1, 3, 5]
c << [2, 4, 6]
```

Reply:
As an User I think it would be easier to understand if we say that we can split _one_ long data table into separate sections. And also include the example you've just made here (excluding the data-pipes)